app/services/background.py

```
@@ -66,40 +66,91 @@ def __init__(self):
+        max_retries = 3
+        retry_delay = 5  # seconds
+        
+        for attempt in range(max_retries):
+            try:
+                self.services[service_name] = "running"
+                self.logger.info(f"✅ {service_name} service started successfully")
+                
+                # Run the service function
+                await service_func()
+                
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)